tests/foreman/ui/test_errata.py

```diff
@@ -834,13 +835,21 @@ def test_positive_list_permission(
 @pytest.mark.e2e
 @pytest.mark.upgrade
 @pytest.mark.no_containers
+@pytest.mark.parametrize(
+    'registered_contenthosts',
+    [  # One test param for each RHEL major version supported
+        pytest.param(
+            {'rhel': rhel_ver, 'count': 3},
+            id=f'rhel{rhel_ver}_{3}reg_chosts',
```

Comment:
is this what you were going for?
```suggestion
            id=f'rhel{rhel_ver}_3reg_chosts',
```

Reply:
yes, I just kept the 3 in brackets so it would be visible, if someone ever changed in the future.
can't find a way to save param 'count' to be used in the param's id, without making it ugly or breaking